rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/util/DefaultImageNameFactorySpec.groovy

```diff
@@ -63,6 +63,64 @@ class DefaultImageNameFactorySpec extends Specification {
       packagesParameter == "nflx-djangobase-enhanced=0.1-3"
   }
 
+  void "should build the imageName based on the ami_name tag"() {
```

Comment:
Nice thorough tests.
Please try to match the formatting of the neighboring tests (the content of the `setup/when/then` clauses is indented 2 spaces).